### PR TITLE
First commit of Web Component

### PR DESCRIPTION
**Pull request by PaulKinlan**
+ Uses microbundle
+ Creates a checked in dist folder
+ Removes CSS
+ Creates an example
+ Uses bind, but it's copied in

### CODE REVIEW

File: dist/filedrop.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
...addEventListener("dragenter",this._onDragEnter),this.addEventListener("dragend",()=>this._reset()),this.addEventListener("dragleave",this._onDragLeave),this.addEventListener("paste",this._onPaste)}get accept(){return this.getAttribute("accept")||""}set accept(t){this.setAttribute("accept",t)}_onDragEnter(t){if(this._dragEnterCount+=1,this._dragEnterCount>1)return;const e=!t.dataTransfer.items.length||!!r(t.dataTransfer.items,this.accept);this.classList.add(e?"drop-valid":"drop-invalid")}_onDragLeave(){this._dragEnterCount-=1,0===this._dragEnterCount&&this._reset()}_onDrop(t){t.preventDefault(),this._reset();const e=n(t.dataTransfer,this.accept);void 0!==e&&this.dispatchEvent(new i("filedrop",{action:"drop",file:e}))}_onPaste(t){const e=n(t.clipboardData,this.accept);void 0!==e&&this.dispatchEvent(new i("filedrop",{action:"paste",file:e}))}_reset(){this._dragEnterCount=0,this.classList.remove("drop-valid"),this.classList.remove("drop-invalid")}}t([e],s.prototype,"_onDragEnter",null),t([e],s.prototype,"_onDragLeave",null),t([e],s.prototype,"_onDrop",null),t([e],s.prototype,"_onPaste",null),customElements.define("file-drop",s),exports.bind=e,exports.FileDropEvent=i,exports.FileDrop=s;
```
Comment (developit) [inline dist/filedrop.js]: Fairly uncommon practise to commit dist files. For the polyfills and other things of similar nature, we've been linking to unpkg for them and gitignoring.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    "element"
+  ],
+  "author": "Paul Kinlan",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/GoogleChromeLabs/file-drop/issues"
+  },
+  "homepage": "https://github.com/GoogleChromeLabs/file-drop#readme",
+  "dependencies": {
+    "microbundle": "^0.6.0"
```
Comment (developit) [inline package.json]: `devDependencies`

File: tslintt.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (developit) [inline tslintt.json]: filename has an extra `t`?

Comment (PaulKinlan) [reply tslintt.json]: ah. good catch.

File: dist/filedrop.js (hunk 1 above)
Comment (PaulKinlan) [reply dist/filedrop.js]: iirc I thought comlink did it.... anyway, removed.

**Pull request merged**